modules/mod_admin/models/m_admin_status.erl

```diff
@@ -63,6 +63,22 @@ m_find_value(unused, #m{value=memory}, _Context) ->
 m_find_value(usage, #m{value=memory}, _Context) ->
     recon_alloc:memory(usage);
 
+% modules
+m_find_value(modules, #m{value=undefined} = M, Context) ->
+    M#m{value=modules};
+m_find_value(down, #m{value=modules}, Context) ->
+    Status = z_module_manager:get_modules_status(Context),
+    lists:flatten([
+        [ Module || {Module, _, _Pid, _Date} <- Specs ]
+        || {State, Specs} <- Status, State =/= running
```

Comment:
Is it not possible to do this with `[Module || {State, {Module, _, _Pid, _Date}} <- Status, State =/= running]`?

Reply:
No, per State we have list, so we end up with a nested list comprehension.